dsl.md

```diff
@@ -101,7 +101,8 @@ Serverless Workflow DSL allows for defining reusable components that can be refe
 The Serverless Workflow DSL defines several default [task](dsl-reference.md#tasks) types that runtimes **must** implement:
 
 - [Call](dsl-reference.md#call), used to call services and/or functions.
-- [Composite](dsl-reference.md#composite), used to define a minimum of two subtasks to perform.
+- [Do](dsl-reference.md#do), used to define a minimum of two subtasks to perform in sequence.
```

Comment:
Shouldn't we be less restrictive for `do` and allow a single task ?

Reply:
yeah I removed it. It's not currently enforced by the schema anyways